lib/model/Fixture.mjs

```diff
@@ -154,6 +154,26 @@ export default class Fixture {
     return this._jsonObject.comment || ``;
   }
 
+  /**
+   * @returns {?string} A string describing the help that is needed for this fixture, or null if no help is needed.
+   */
+  get helpWanted() {
+    return this._jsonObject.helpWanted || null;
+  }
+
+  /**
+   * @returns {!boolean} True if help is needed in this fixture (maybe in a capability), false otherwise.
+   */
+  get isHelpWanted() {
+    if (!(`isHelpWanted` in this._cache)) {
+      this._cache.isHelpWanted = this.helpWanted !== null || this.capabilities.some(
```

Comment:
Why not `this.availableChannels.concat(this.templateChannels).some(ch => ch.isHelpWanted)`? No need to check all capabilities if the channels already do this.